bycycle/cyclepoints.py

```diff
@@ -292,3 +261,30 @@ def extrema_interpolated_phase(sig, ps, ts, zerox_rise=None, zerox_decay=None):
     pha_tnpi[-last_empirical_idx + 1:] = np.nan
 
     return pha_tnpi
+
+
+def find_flank_zero_xs(sig, flank):
```

Comment:
So I refactored most of this file a bunch, and made sub-functions. But I didn't really solve the naming, as all the sub-functions ended up with sorta the same name (since they seem to have quite overlapping function). I think a ToDo is to logic through this file again, and check and probably update a naming scheme here.

Reply:
It was difficult to find names to differenitate them. Updated in 75d9945. These aren't great still, however.

`find_flank_zero_xs -> find_flank_zerox`
I renamed this to keep it consistent with the other zerox funcs.

`_find_zerox_flanks -> _find_flank_midpoints`
I renamed this to differentiate it from other zerox funcs. This function finds location of the median
voltage between two extrema, rather than where the voltage == 0 as in find_flank_zerox.

After renaming this, I guess `find_zerox` is kind of a misnomer since it only returns zero-crossings
for cycles that are perfectly symetrical, which is unlikely in real data.